### PR TITLE
Add ui to docker compose example

### PR DESCRIPTION
**Pull request by edbo**
Minimal config to get ui working

### CODE REVIEW

**Review by 31z4 — CHANGES_REQUESTED** [review]
Please also add a note regarding the UI below the compose example.

**Comment by edbo** [comment]
OK I've made a small update to the notes as well now.

File: README.md [hunk 1]
```diff
@@ -79,9 +79,23 @@ services:
...
+            - nimbus
+        restart: always
+        ports:
+            - 8080:8080
 ```
 
-Run `docker-compose up` and wait for it to initialize completely. The Nimbus will be available at your host and port `6627`.
+Run `docker-compose up` and wait for it to initialize completely. The Nimbus will be available at your host and port `6627` and the UI available at port `8080` through your browser.
+
+Note: the supervisor logs will not currently work correctly in the UI.
```
Comment (31z4) [inline README.md]: Actually neither of logs will work from the UI.

Comment (edbo) [reply README.md]: OK I've made this more general instead.

**Pull request closed without merging**